composites/Plugin/Shared/components/StackedProgressBar.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const StackedProgressBarContainer = styled.div`
+	box-sizing: border-box;
+	height: ${ props => props.height };
+	overflow: hidden;
+`;
+
+StackedProgressBarContainer.propTypes = {
+	height: PropTypes.string,
+};
+
+StackedProgressBarContainer.defaultProps = {
+	height: "40px",
+};
+
+const StackedProgressBarProgress = styled.span`
+	box-sizing: border-box;
+	display: inline-block;
+	width: ${ props => `${ props.progressWidth }%` };
+	background-color: ${ props => props.progressColor };
+	height: 100%;
+`;
+
+StackedProgressBarProgress.propTypes = {
+	progressWidth: PropTypes.number.isRequired,
+	progressColor: PropTypes.string.isRequired,
+};
+
+const StackedProgressBar = ( props ) => {
+	let totalValue = 0;
+	for( let i = 0; i < props.items.length; i++ ) {
+		totalValue += props.items[ i ].value;
+	}
+	return(
+		<StackedProgressBarContainer
+			aria-hidden="true" >
```

Comment:
I think we don't need aria-hidden here. If I'm not wrong, these are just div/span elements with some background. Instead, in other cases where we use a `<progress>` element, we hide it because it's not well supported yet for our use case (max value different from 100).